### ISSUE TITLE
The Distros list REALLY needs some cleanup / update

### ISSUE THREAD

**Issue by bmferreira** [unetbootin/unetbootin#83]

**Comment by Cravix** [comment]
yeah, arch linux distro build has been broken for a long time, but I also doubt if unetbootin project is actually deprecated...

**Comment by gkovacs** [comment]
not deprecated I am simply rather busy and distros update too often with inconsistent URL patterns so it is a maintenance nightmare. may perhaps just remove the download feature. but if anyone would like to update the distros list I will gladly take a pull request, it is all in https://github.com/unetbootin/unetbootin/blob/master/src/unetbootin/distrover.cpp and https://github.com/unetbootin/unetbootin/blob/master/src/unetbootin/distrolst.cpp

**Comment by Cravix** [comment]
eh sorry for not being clear :( I didn't mean doing some cleanup (that doesn't really matter to me tbh), just saying some can't be built now and needs some update, 
like when I feed archlinux-2016.09.03-dual.iso into unetbootin, the live usb it generates can't really do its work, to be exact, can't display the correct boot menu.
hope that helps, and thanks for your work :)

**Comment by JtLively** [comment]
@gkovacs how would I test my update to this ? Edit a specific file for the program on my desktop ?

**Comment by genbtc** [comment]
Bump. this needs fixed. WE need a link to Kali @ https://www.kali.org/ instead.

**Comment by asmodehn** [comment]
Sounds like unetbootin needs a feature to discover interactively downloadable isos from a URL, instead of listing them all... Would probably help with the maintenance.

**Comment by JtLively** [comment]
@asmodehn That would be great - maybe an auto-detect for .iso's of a specified folder. I always download anyway - easier to test the md5sum and ensure I have the newest correct version. 
Thanks, by the way for responding to this. You reminded me I never released my How-to video for unetbootin. I plan to work on [Etcher](https://etcher.io/) next.
